liquibase-core/src/main/java/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParser.java

```diff
@@ -319,7 +342,40 @@ public DatabaseChangeLog parse(String physicalChangeLogLocation, ChangeLogParame
         return changeLog;
     }
 
-
+    private void handleProperty(ChangeLogParameters changeLogParameters, DatabaseChangeLog changeLog, Matcher propertyPatternMatcher) {
+        String name = null;
+        String value = null;
+        String context = null;
+        String labels = null;
+        String dbms = null;
+        boolean global = false;
+        for (int i = 1; i <= propertyPatternMatcher.groupCount(); i++) {
+            String temp = propertyPatternMatcher.group(i);
+            String[] parts = temp.split(":");
+            String key = parts[0].trim();
```

Comment:
Should you do `.toLowerCase()` on the `key` since the regular expression is case insensitive?

Reply:
In this case, the container class, `ChangeLogParameters` does a case-insensitive lookup.

Reply:
Right, but if someone types in the key as `nAmE` for example, the regex would match on it, but this switch statement would not. Unless I'm misunderstanding something. Or perhaps entering `nAmE` is not valid.

Reply:
My bad.  I misunderstood which key.  You are exactly right.